spec/acceptance/self_service_spec.rb

```diff
@@ -235,6 +235,10 @@ class {'self_service':
         run_shell('systemctl start puppet_system_processes-metrics.timer')
         run_shell('puppet agent --enable')
       end
+      it 'if S0027 conditions for false are met' do
+        result = run_shell('facter -p self_service.S0027')
+        expect(result.stdout).to match(%r{false})
+      end
```

Comment:
this acceptance test does not have failure conditions so all acceptance will fail, Jarret had a suggestion, but it would be cleaner not to test than to have a failure test that can not be triggered